### ISSUE TITLE
Laravel 6 please

### ISSUE THREAD

**Issue by tanthammar** [mxl/laravel-queue-rate-limit#3]
Tried to create PR for Laravel 6 but get an error on the Worker.
__construct needs a `callable $isDownForMaintenance`

```
parent::__construct($manager, $events, $exceptions);
```
any suggestions?

```
/**
     * Create a new queue worker.
     *
     * @param  \Illuminate\Contracts\Queue\Factory $manager
     * @param  \Illuminate\Contracts\Events\Dispatcher  $events
     * @param  \Illuminate\Contracts\Debug\ExceptionHandler  $exceptions
     * @param  \callable $isDownForMaintenance
     * @return void
     */
    public function __construct(QueueManager $manager,
                                Dispatcher $events,
                                ExceptionHandler $exceptions,
                                callable $isDownForMaintenance)
    {
        $this->events = $events;
        $this->manager = $manager;
        $this->exceptions = $exceptions;
        $this->isDownForMaintenance = $isDownForMaintenance;
    }
```

**Comment by hettiger** [comment]
I got it running on Laravel 6: #2

**Comment by mxl** [comment]
Closed in #2.

**Comment by mxl** [comment]
Laravel 6 support is available in version 2.0.0. Thank you again for your job!
